### DISCUSSION TITLE
windows_eventlog tuning

### DISCUSSION THREAD

**Discussion by 5MI7th3MI6** [Security-Onion-Solutions/securityonion#9388] [Unsupported Versions]
Hello, 

I am getting a lot of general windows_eventlogs for alerts, any idea of how I can tune them? 

![image](https://user-images.githubusercontent.com/68974383/207469455-471a07c6-911c-40b0-8fd6-67452a03cf77.png)

**Comment by dougburks** [comment]
Depends on how your Windows logs are coming in:
https://docs.securityonion.net/en/2.3/host.html
